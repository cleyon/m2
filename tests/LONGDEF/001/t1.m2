@longdef FOO
FOO 1
FOO 2
FOO 3
@endlong
FOO is >@FOO@<
