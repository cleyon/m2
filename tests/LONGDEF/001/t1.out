FOO is >FOO 1
FOO 2
FOO 3<
